lib/private/Profile/TProfileHelper.php

```diff
@@ -26,19 +26,12 @@
 
 namespace OC\Profile;
 
-use OCP\Accounts\IAccount;
-use OCP\Accounts\IAccountManager;
+use OCP\IConfig;
 
 trait TProfileHelper {
-
-	/**
-	 * Returns whether the profile is enabled for the account
-	 *
-	 * @since 23.0.0
-	 */
-	protected function isProfileEnabled(IAccount $account): ?bool {
+	protected function isProfileEnabledByDefault(IConfig $config): ?bool {
```

Comment:
Will always be a bool, no?

```suggestion
	protected function isProfileEnabledByDefault(IConfig $config): bool {
```

Reply:
`FILTER_NULL_ON_FAILURE` is used for `filter_var()` to be more strict though only truthiness is checked when this is called so this is mainly to distinguish between expected and unexpected values for debugging purposes should it be needed in the future

Reply:
Not sure that I understand the rational, but ok, I trust your judgment ;)